src/apps/answers/service.py

```diff
@@ -568,6 +585,67 @@ async def get_activity_answer(
         )
 
         return submission
+    
+    async def _validate_user_role_for_take_now(
+            self, 
+            applet_id: uuid.UUID, 
+            respondent: SubjectSchema
+    ) -> None:
+        is_user_role_restrictive = await UserAppletAccessCRUD(self.session).get_by_roles(
+            respondent.user_id,
+            applet_id,
+            # Define a list of roles prohibited from accessing the applet            
+            [Role.SUPER_ADMIN, Role.EDITOR, Role.COORDINATOR, Role.REVIEWER]
+        )
```

Comment:
Role.managers() is already that list

Reply:
Use same call you have in validate_multiinformant_assessment()
```
       is_admin = await AppletAccessCRUD(self.session).has_any_roles_for_applet(
            respondent_subject.applet_id,
            respondent_subject.user_id,
            Role.managers(),
        )
```

Reply:
@rcmerlo I believe this check is correct and necessary to prevent team members with the listed roles from performing take now. Using `Role.managers()` would produce the wrong list of roles here.

The only thing I would change here is removing `Role.SUPER_ADMIN` from this list of restricted roles

Reply:
`Role.managers` has the following values associated with it
```python
[
            cls.OWNER,
            cls.MANAGER,
            cls.COORDINATOR,
            cls.EDITOR,
            cls.REVIEWER,
        ]
```
the `MANAGER` is not a prohibited role and we want to allow this role to access so I would assume that using this  list will not do the work in here unless I am mistaken